lib/tsclient/client.rb

```diff
@@ -26,6 +26,15 @@ def whois(addr)
       end
     end
 
+    def status
+      response = api_get(:status)
+      if response.error?
+        nil
```

Comment:
Mostly musing for future thoughts, we should probably raise an error to the user with the error info in or expose a wrapper object that exposes what the error is.

This is totally fine for now though, happy the user can check for `nil`, they just can't see what the error actually is without further investigation.

Reply:
Just noticed you've copied the pattern from the existing code actually, which is *awesome* to see. Apologies for me criticising my own code via yours 😛

Reply:
😆 all good!